### ISSUE TITLE
add return super().coords(tag_or_id, *args) in ctk canavas please

### ISSUE THREAD

**Issue by MaxNiark** [TomSchimansky/CustomTkinter#2066]
To get the return of coord like tkinter basic 

thanks 

![image](https://github.com/TomSchimansky/CustomTkinter/assets/126392328/78344b20-f026-48bb-ad34-43bf8fa50540)

**Comment by TomSchimansky** [comment]
CTkCanvas is not meant to be used outside the library itself, its just written for rendering the CTk components.

**Comment by MaxNiark** [comment]
yeah, i'm using this for something else in my app, i use a canvas widget for matplotlib :)
